proto/server.proto

```diff
@@ -140,6 +140,37 @@ message SignUpRequest {
   Meta meta = 5;
 }
 
+message CreateFolderRequest {
+  string path = 1;
+  string name = 2;
+  Meta meta = 3;
+}
+message ListFilesRequest {
+  string path = 1;
+  Meta meta = 2;
+}
+message DeleteFileRequest {
+  string path = 1; // Full path, including the file name and extension
+  Meta meta = 2;
+}
+message DownloadFileRequest {
+  string path = 1; // Full path, including the file name and extension
+  Meta meta = 2;
+}
+message UploadFileRequest {
+  string path = 1;
+  string name = 2;
+  bytes payload = 3;
+  int64 size = 4; // Size of the whole file in bytes
+  string hash = 5; // Hash for the whole file
+  Meta meta = 6;
+}
+message FilePayload {
```

Comment:
Which fields will be present in the subsequent payloads?

Reply:
Only the payload.
Do you want to keep a uuid also as a field?

Reply:
nope. The payload is fine

Reply:
Ok